help_to_heat/frontdoor/schemas.py

```diff
@@ -44,7 +44,6 @@
     epc_rating_field,
     epc_select_manual_page,
     epc_select_page,
-    field_dont_know,
```

Comment:
Is this unused in the constants file now?

Reply:
I don't believe so, there's other questions that use this field in the consts.py file, as like "Yes" and "No" it's sort of a generic answer:
 `wall_insulation_field_dont_know = field_dont_know`

This import in particular in schemas was unused once I removed its usage below, though.